CMakeLists.txt

```diff
@@ -1,21 +1,23 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 2.6.2)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.1.0 FATAL_ERROR)
```

Comment:
this will limit backporting the app for older distributions, which we do for openSUSE e.g. with the OBS the nitrokey project could do the same easily for debian/ubuntu/fedora/rhel/arch as well.